### PR TITLE
feat: add icons in experiment and trial detail pages

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
WEB-681

Related PR: #5373

## Screenshots

image

image

## Test Plan
- Check if icons are visible in experiment and trial detail pages
- Check if buttons are actionable (pause, stop etc) if there are actionable buttons
- Check if non-actionable buttons show tooltip by hover
- Check if actionable buttons don't show tooltip by hover

## Checklist
- [x] Changes have been manually QA'd
- [x] User-facing API changes need the "User-facing API Change" label.
- [x] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/components/ExperimentIcons/index.tsx [hunk 1]
```diff
@@ -37,14 +40,15 @@ const ExperimentIcons: React.FC<Props> = ({ state }) => {
...
         return <Active />;
       default:
         return <Icon name="cancelled" style={{ ...IconStyle, color: 'var(--theme-ix-cancel)' }} />;
     }
   }, [state]);
   return (
-    <Tooltip title={runStateToLabel[state]}>
-      <span>{icon}</span>
+    <Tooltip title={stateToLabel(state)}>
+      <span style={{ display: 'inherit' }}>{icon}</span>
```
Comment (keita-determined) [inline webui/react/src/components/ExperimentIcons/index.tsx:51]: added `display: 'inherit'` for alignment

**Comment by ClaireNeveu** [comment]
Can you add a screenshot to this PR?

**Comment by keita-determined** [comment]
> Can you add a screenshot to this PR?

added in description

**Review by gt2345 — COMMENTED** [review]
Screen Shot 2022-11-30 at 2 51 39 PM
3 icons next to each other may look a little off to me, but it's best to consult Alex on this.

**Comment by keita-determined** [comment]
Talked to Alex and will change some code.

File: webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx [hunk 2]
```diff
@@ -21,6 +22,7 @@ const TrialHeaderLeft: React.FC<Props> = ({ experiment, trial }: Props) => {
       </Link>
       <Icon name="arrow-right" size="tiny" />
       <div className={css.trial}>
+        {<ExperimentIcons state={trial.state} />}
```
Comment (gt2345) [inline webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx:25]: nit: remove the bracket

Comment (keita-determined) [reply webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx:25]: nice catch

**Comment by keita-determined** [comment]
https://user-images.githubusercontent.com/109113616/204945131-ea0de907-5e2c-44d7-9658-9118d11b612e.mov

File: webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx [hunk 3]
```diff
@@ -33,13 +34,67 @@ import Spinner from 'shared/components/Spinner/Spinner';
...
+  TrialItem,
+} from 'types';
 import handleError from 'utils/error';
 import { canActionExperiment, getActionsForExperiment } from 'utils/experiment';
 import { openCommandResponse } from 'utils/wait';
 
 import css from './ExperimentDetailsHeader.module.scss';
 
+// Actionable means that user can take an action, such as pause, stop
+const isActonableIcon = (state: CompoundRunState): boolean => {
```
Comment (gt2345) [inline webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx:52]: typo

**Review by gt2345 — APPROVED** [review]
Looks very nice to me

**Comment by gt2345** [comment]
Forgot to mention, I recommend mark this PR as feat rather than chore because it's user facing

File: webui/react/src/components/ExperimentIcons/Queue.module.scss [hunk 4]
```diff
@@ -1,18 +1,24 @@
 .base {
-  align-items: center;
-  display: flex;
   height: 24px;
-  justify-content: center;
+  margin: auto;
```
Comment (keita-determined) [inline webui/react/src/components/ExperimentIcons/Queue.module.scss:3]: i changed Queue scss to make it resizable.

File: webui/react/src/shared/themes.ts [hunk 5]
```diff
@@ -153,8 +153,8 @@ const themeBase = {
   statusWarningOnStrong: undefined,
   statusWarningOnWeak: undefined,
 
-  breathButton: 'rgba(143, 143, 143, 0.4)',
-  breathButtonOverlay: 'rgba(143, 143, 143, 0.2)',
+  breathButton: 'rgba(143, 143, 143, 0.7)',
+  breathButtonOverlay: 'rgba(143, 143, 143, 0.4)',
```
Comment (keita-determined) [inline webui/react/src/shared/themes.ts:157]: higher opacity for visibility improvement

File: webui/react/src/shared/themes.ts [hunk 6]
```diff
@@ -152,9 +152,6 @@ const themeBase = {
   statusWarningOn: 'rgba(255, 255, 255, 1.0)',
   statusWarningOnStrong: undefined,
   statusWarningOnWeak: undefined,
-
-  breathButton: 'rgba(143, 143, 143, 0.4)',
-  breathButtonOverlay: 'rgba(143, 143, 143, 0.2)',
```
Comment (keita-determined) [inline webui/react/src/shared/themes.ts:157]: these are used only in `Queue`, so moved them to the `Queue` scss.
@gt2345 is it fine? I can revert it if these could be used in the future

Comment (gt2345) [reply webui/react/src/shared/themes.ts:157]: That's fine~

**Review by gt2345 — APPROVED** [review]

**Pull request merged**